osh.spec

```diff
@@ -234,19 +234,40 @@ fi
 %{_sbindir}/osh-retention
 %{_sbindir}/osh-stats
 %{_sysconfdir}/osh/hub
+%exclude %{python3_sitelib}/osh/hub/scripts/osh-xmlrpc-client.py
+%exclude %{python3_sitelib}/osh/hub/scripts/umb-emit.py
+%attr(755,root,root) %{python3_sitelib}/osh/hub/scripts/osh-xmlrpc-client.py
+%attr(755,root,root) %{python3_sitelib}/osh/hub/scripts/umb-emit.py
```

Comment:
I do not think we have ever used these scripts from an installed RPM package.  I would just remove them in `%install`.